### PR TITLE
Azure sentinel incident url

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/44026

## Description
Added the `IncidentUrl` output to the `azure-sentinel-get-incident-by-id` command.

## Minimum version of Cortex XSOAR
- [x] 6.0.0

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

**Comment by Shellyber** [comment]
Validation fails on missing output in README, but I've added it to the command

File: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.yml [hunk 1]
```diff
@@ -145,11 +144,14 @@ script:
     - contextPath: AzureSentinel.Incident.Etag
       description: The Etag of the incident.
       type: String
+    - contextPath: AzureSentinel.Incident.IncidentUrl
+      description: The link url to the incident in Azure portal.
```
Comment (Itay4) [inline Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.yml:148]:
```suggestion
      description: The deep-link url to the incident in Azure portal.
```

File: Packs/AzureSentinel/ReleaseNotes/1_2_3.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Azure Sentinel
+- Added the **IncidentUrl** output to **azure-sentinel-get-incident-by-id** command.
```
Comment (Itay4) [inline Packs/AzureSentinel/ReleaseNotes/1_2_3.md:4]:
```suggestion
- Added the **IncidentUrl** output to the **azure-sentinel-get-incident-by-id** command.
```

**Review by Itay4 — COMMENTED** [review]
please make the url clickable in the readable output

**Comment by Shellyber** [comment]
@Itay4 All done

File: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py [hunk 3]
```diff
@@ -405,7 +406,9 @@ def get_incident_by_id_command(client, args):
 
     result = client.http_request('GET', url_suffix)
     incident = incident_data_to_xsoar_format(result)
-    readable_output = tableToMarkdown(f'Incident {inc_id} details', incident,
+    content = incident.copy()
+    content['IncidentUrl'] = create_clickable_url(content.get('IncidentUrl'))
+    readable_output = tableToMarkdown(f'Incident {inc_id} details', content,
```
Comment (Itay4) [inline Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py:411]: i think you can use `url_keys` instead of `create_clickable_url`

Comment (Shellyber) [reply Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py]: Right. Wasn't familiar with this. Updated!

**Review by Itay4 — APPROVED** [review]
looks good

**Pull request merged**